include/mouse.h

```diff
@@ -228,6 +231,8 @@ class MouseControlAPI final {
 private:
 	MouseControlAPI(const MouseControlAPI &)            = delete;
 	MouseControlAPI &operator=(const MouseControlAPI &) = delete;
+
+	bool was_interactive_mapping_started = false;
```

Comment:
`was_interactive_mapping_started` is is super self-documenting; excellent! 

In relation to the state's `is_mapping_in_progress` bool, are they roughly the same - but cover slightly different moments in time?

Not asking for any changes here, but just pointing it out (if you see any overlap).

Reply:
They cover the same moment. I just want to know, that this particular `MouseControlAPI` object is the one which is used to perform the command-line interactive mapping - as, once the GUI is implemented, it will be possible that two such objects will exist at the same time.